pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,110 @@
+    * Unintentional discrimination of research participants, such as different perception of symptom severity in different population groups
+    * Findings about participants that are outside of the original research question(s): see [Purpose of data processing](https://commission.europa.eu/law/law-topic/data-protection/reform/rules-business-and-organisations/principles-gdpr/purpose-data-processing_en) and [Incidental findings Archives - BBMRI-ERIC](https://www.bbmri-eric.eu/elsi-topic/incidental-findings/)
+​
+* Other ethical questions are arising from the impact of research outcomes, including data on the interest of communities or individuals
+    * Fair management of intellectual property rights - also see e.g. Access and Benefit-sharing (ABS)
+    * Publication of research data that might impact the reputation of communities or individuals
+    * Publication of research data that might impact economical interests of  communities or individuals
+    * Publication of research data that might impact security of society, communities or individuals
+​
+* There are also general research ethics considerations that are relevant in the context of research data, including:
+    * What are the reasons justifying the exclusion/inclusion of research data in a particular context?
+    * Is the data source accurate and trustworthy?
+    * How can bias in practices of research data management be identified and minimised/avoided?
+    * Assessment of models and algorithms used with respect to possible bias
+    * Can the research data be misinterpreted?
+    * Prevention of withholding of research data
+    * Prevention of manipulation and fraud of research data
+    * Assessment of who is excluded or included to data access and why
+    * How can harm to other beings and the environment be identified and mitigated in a timely manner?
+​
+### Solutions
+* Assess potential ethical implication through an ethics review
+    * Your local ethics committee can help you to review the ethical implications of the project or might guide you to more relevant bodies and resources
+* In order to address challenges when working with human data (see also the [RDMkit page on human data](human_data))
+    * Used standardised consent forms (see e.g. [GA4GH consent toolkit]({% raw %}https://www.ga4gh.org/resource-library/#{%22document%22:{%22category%22:%22Toolkit%20Component%22,%22related_products%22:%22Consent%20Toolkit%22}}{% endraw %}), [BBMRI](https://zenodo.org/record/3928446)) that can be represented in a machine actionable way ([DUO](https://www.ga4gh.org/product/data-use-ontology-duo/))
```

Comment:
This point needs to revised, including the link to consent toolkit - it's not displayed properly in preview. Also, there needs to be some more/clear information in order to mention DUO and the GA4GH link for it.

Reply:
```suggestion
    * Used standardised consent forms (see e.g. 
{% tool "ga4gh-regulatory-and-ethics-toolkit" %}) with clauses that can be represented in a machine actionable way using the {% tool "data-use-ontology" %} and {% tool "informed-consent-ontology" %})
```